ASTree.cpp

```diff
@@ -1363,19 +1402,9 @@ PycRef<ASTNode> BuildFromCode(PycRef<PycCode> code, PycModule* mod)
                     break;
                 }
 
-                if ((curblock->blktype() == ASTBlock::BLK_WHILE
-                            && !curblock->inited())
-                        || (curblock->blktype() == ASTBlock::BLK_IF
-                            && curblock->size() == 0)) {
-                    PycRef<PycObject> fakeint = new PycInt(1);
-                    PycRef<ASTNode> truthy = new ASTObject(fakeint);
-
-                    stack.push(truthy);
-                    break;
-                }
```

Comment:
Why is this block removed?  It doesn't look related to the change...

Reply:
This was "fake" (fakeint, truthy) handling of the same use case. Should be removed.

Reply:
Hmm, looking at it, I assumed it was for cases like `if True` or `while 1`, but I can't seem to trigger it...  So I'll assume you're right, since I don't remember the origin of this code :)